test/test_automl/test_estimators.py

```diff
@@ -62,6 +62,7 @@ def __call__(self, *args, **kwargs):
     get_smac_object_wrapper_instance = get_smac_object_wrapper()
 
     automl = AutoSklearnClassifier(
+        delete_tmp_folder_after_terminate=False,
```

Comment:
Why is this now necessary?

Reply:
This is because after fit there is no need for the backend to survive, yet the test still wants to go to the temporary directory and check for the correctness of some files.